site/content/release-guide.md

```diff
@@ -239,8 +248,10 @@ The release tarball, signature, and checksums are here:
 
 Helm charts are available on:
 * https://dist.apache.org/repos/dist/dev/incubator/polaris/helm-chart
-NB: the docker images (polaris-server and polaris-admin) will be
-published on DockerHub once release vote passes.
+
+Docker images:
```

Comment:
Since those are multiplatform images, probably worth to add a link to Dockerhub

Reply:
Good point, I will.